### PR TITLE
feat(sdk-coin-atom): add keyPair management to cosmos

### PR DESCRIPTION
**Pull request by michael-gallegos-bg**
TICKET: BG-66310

### CODE REVIEW

File: modules/sdk-coin-atom/src/lib/keyPair.ts [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  }
+
+  /**
+   * Cosmos default keys format: raw private and compressed public key
+   *
+   * @returns { DefaultKeys } The keys in the protocol default key format
+   */
+  getKeys(): DefaultKeys {
+    return {
+      pub: this.getPublicKey({ compressed: true }).toString('hex'),
```
Comment (michael-gallegos-bg) [inline modules/sdk-coin-atom/src/lib/keyPair.ts:53]: > A compressed format is used for secp256k1 and secp256r1 serialization.
https://docs.cosmos.network/v0.46/basics/accounts.html

Need to double check if this is what is wanted

File: modules/sdk-coin-atom/src/lib/keyPair.ts [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+import {
+  DefaultKeys,
+  isPrivateKey,
+  isPublicKey,
+  isSeed,
+  KeyPairOptions,
+  Secp256k1ExtendedKeyPair,
+} from '@bitgo/sdk-core';
+
+const DEFAULT_SEED_SIZE_BYTES = 16;
```
Comment (Vijay-Jagannathan) [inline modules/sdk-coin-atom/src/lib/keyPair.ts:14]: Is this common across chains (does seem like it but confirming) ? If yes, do we need to put it somewhere in the `Secp256k1ExtendedKeyPair` class or any other appropriate file ?

File: modules/sdk-coin-atom/src/lib/keyPair.ts (hunk 1 above)
Comment (Vijay-Jagannathan) [reply modules/sdk-coin-atom/src/lib/keyPair.ts:53]: While I was trying to create a wallet address using the `@cosmjs/amino` library, it complained that I was having an `Invalid Secp256k1 pubkey length (compressed): 48` and that the correct compressed length was 33 bytes - [Link 1](https://github.com/cosmos/cosmjs/blob/main/packages/amino/src/pubkeys.ts#L42-L53), [Link 2](https://github.com/cosmos/cosmjs/blob/main/packages/amino/src/addresses.ts#L17). From this I believe compressed is what's expected but I'll defer to another pair of eyes.

File: modules/sdk-coin-atom/src/lib/keyPair.ts (hunk 2 above)
Comment (michael-gallegos-bg) [reply modules/sdk-coin-atom/src/lib/keyPair.ts:14]: You are correct, it seems most commonly 16, but some chains have it at 64.
For our purposes, I will set it in a coin constants file.

**Review by Vijay-Jagannathan — APPROVED** [review]

**Review by sachushaji — APPROVED** [review]

**Pull request merged**